atlassian/__init__.py,sha256=NZKiK4b1IGScaCBh26Q0SGR8SrUf0QzaftAJO6kwHK4,106
atlassian/bitbucket.py,sha256=J_nzkH6-G84iZvHud6eb-GKvMf3CSG_NzHp2cSw0bGc,7957
atlassian/client.py,sha256=KQVf6w_rYE-tJiBuIMJ5zYIOEIaO6deDPnq3r7pmMbg,2781
atlassian/error.py,sha256=3rm_uYUaGggiNos0n70Mnb8LSU5tffINZJrCt6NUct8,1669
atlassian/jira.py,sha256=PHuEi_POc3cdnVCyfi5_W8-b0FXxGqfQzfNR0aTAXuo,9311
atlassian/logger.py,sha256=ZGoIeINuzVqpjpVdTvE_9U7qrbvdS4qw89KtdJ4XQjE,1164
atlassian_api_py-0.0.8.dist-info/LICENSE,sha256=AXit2T0JlqTMEDMfTMN_i0muiJOTtPq3s89gooVOsEk,1069
atlassian_api_py-0.0.8.dist-info/METADATA,sha256=3-SmvlZSqXH1l8gWQBzlLYL0ORdtjBbTvfRGqQP0KhI,2190
atlassian_api_py-0.0.8.dist-info/WHEEL,sha256=YUYzQ6UQdoqxXjimOitTqynltBCkwY6qlTfTh2IzqQU,97
atlassian_api_py-0.0.8.dist-info/top_level.txt,sha256=84-EDv8LNaZx01TCGjBzglwPU1JSRJ7fPSq9uy55Sv0,10
atlassian_api_py-0.0.8.dist-info/RECORD,,
